spec/models/admin_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Admin, type: :model do
+=begin
+  # undefined method `encrypted_password=' for #<Admin:0x000000033dda80>
+  it 'should be valid' do
+    build(:admin).should be_valid
```

Comment:
@rakvium db:reset db:migrate didn't help. Error is connected with devise :database_authenticatable. Test passed without raising error after I removed :database_authenticatable from model.